File: Middleware/src/main/java/Shared/Ticket.java
package Shared;

import java.io.Serializable;

public class Ticket implements Serializable
{
public int price;
public String luggage;
public int seat;

  public Ticket(int price, String luggage)
  {
    this.price = price;
    this.luggage = luggage;
  }

  public Ticket(int price, String luggage, int seat)
  {
    this.price = price;
    this.luggage = luggage;
    this.seat = seat;
  }

  public Ticket()
  {

  }

  public boolean equals(Object obj)
  {
    if (!(obj instanceof Ticket))
      return false;
    Ticket other = (Ticket) obj;
    return price==other.price&&luggage.equals(other.luggage)&&seat== other.seat;
  }

  @Override public String toString()
  {
    return "Ticket{" + "price=" + price + ", luggage='" + luggage + '\'' + '}';
  }
}

File: Client/Client/Model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Client.Model
{
    public class Ticket
    {
        public Ticket(int price, string luggage)
        {
            this.price = price;
            this.luggage = luggage;
        }

        public Ticket(int price, string luggage,int seat)
        {
            this.price = price;
            this.luggage = luggage;
            this.seat = seat;
        }

        public Ticket()
        {
        }

        [JsonPropertyName("price")]
        public int price { get; set; }
        [JsonPropertyName("luggage")]
        public string luggage { get; set; }
        [JsonPropertyName("seat")]
        public int seat { get; set; }

        public override string ToString()
        {
            return "Price: " + price + " Luggage: " + luggage;
        }
    }
}
